easybuild/easyblocks/g/gromacs.py

```diff
@@ -182,6 +182,11 @@ def sanity_check_step(self):
         if self.toolchain.options.get('usempi', None):
             suff = '_mpi'
 
+        # Add the _d suffix to the suffix, in case of the double precission
+        if '-DGMX_DOUBLE=on' in self.cfg['configopts']:
+            suff = suff+'_d'
```

Comment:
@paderijk just to be really picky w.r.t. style: please add spaces around `+` for readability, and avoid duplicate empty lines below (i.e. drop on empty line)

If you just make a new commit with those changes and push it to the same branch `gromacs_dp`, this pull request will be updated (i.e. no need to open a new PR).